hardware/sensehat/README.md

```diff
@@ -71,7 +71,9 @@ Joystick events are sent when the Sense HAT joystick is interacted with. The
 This node sends commands to the 8x8 LED display on the Sense HAT.
 
 Commands are sent to the node in `msg.payload`. Multiple commands can
-be sent in a single message by separating them with newline (\n) characters.
+be sent in a single message by separating them with newline (\n) characters. 
+You must use a function node or a change node(jsonata expression) when 
+using the newline (\n) character to have multiple commands
```

Comment:
maybe ?...  to create a payload containing multiple commands.

Reply:
sure